src/metrics/nom.rs

```diff
@@ -113,12 +153,41 @@ impl Nom for JavascriptCode {
         use Javascript::*;
 
         match node.object().kind_id().into() {
-            Function | FunctionDeclaration | MethodDefinition => {
+            FunctionDeclaration | MethodDefinition => {
                 stats.functions += 1;
             }
-            GeneratorFunction | GeneratorFunctionDeclaration | ArrowFunction => {
+            GeneratorFunction | GeneratorFunctionDeclaration => {
                 stats.closures += 1;
             }
+            Function => {
```

Comment:
It's a pain that we have to duplicate this code for mozjs and for js!
Are there actually differences or can we share the same implementation?